x-pack/plugins/reporting/server/types.ts

```diff
@@ -39,13 +39,15 @@ import { ReportingCore } from './core';
  * Plugin Setup Contract
  */
 export interface ReportingSetup {
+  registerExportTypes: ExportTypesRegistry['register'];
+  getEsClient: ReportingCore['getEsClient'];
+  getDataService: ReportingCore['getDataService'];
```

Comment:
Let's figure out how to not have our `getEsClient ` and `getDataService ` exposed in the setup contract